gnovm/pkg/gnolang/gnodebug/debug_enabled.go

```diff
@@ -0,0 +1,34 @@
+//go:build debug
+
+package gnodebug
+
+import (
+	"os"
+	"sync"
+
+	// Enable pprof.
+	//nolint:gosec
+	_ "net/http/pprof"
+)
+
+const Debug DebugType = true
+
+var flagsOnce = sync.OnceValue[DebugFlags](func() DebugFlags {
+	return ParseFlags(os.Getenv("GNODEBUG"))
+})
```

Comment:
nit : Could we not also support direct flag ? `-debug`

Reply:
I think it's easier to control this way, simply because no data has to be passed down from cmd/gno, and setting the env var is more consistent with what happens in go, for instance